zio-kafka/src/main/scala/zio/kafka/utils/ExtraZStreamOps.scala

```diff
@@ -0,0 +1,40 @@
+package zio.kafka.utils
+
+import zio._
+import zio.stream._
+
+object ExtraZStreamOps {
+
+  implicit class ZStreamOps[R, E, A](val stream: ZStream[R, E, A]) {
+
+    /**
+     * Fails the stream with given error if it is not consumed (pulled) from, for some duration.
+     *
+     * Also see [[zio.stream.ZStream#timeoutFail]] for failing the stream doesn't _produce_ a value.
```

Comment:
```suggestion
     * Also see [[zio.stream.ZStream.timeoutFail]] for failing the stream doesn't _produce_ a value.
```

So that we can CTRL+click on it to access the method directly

Reply:
It has to be `#` for scala 2. In Scala 3 you can also use `.`.

Reply:
I'm using Scala 2 and `#` doesn't work for me 🤔

Reply:
Hmmm. It doesn't work for me either. The documentation must be wrong :(
I'll change it.